models/messages/group.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * @file Declares Group data type representing group of users
+ */
+import User from "../users/user";
+import mongoose from "mongoose";
+
+/**
+ * @typedef Group Represents group of users
+ * @property {User[]} users in the group
+ * @property {Date} createdOn date on which the group was created
+ */
+export default interface Group {
+    _id?: mongoose.Schema.Types.ObjectId,
+    name: string
+    users: User[],
```

Comment:
Should we add a field called `owner` here? This will enable us to have this functionality that only the owner of the group can delete the group. What do you guys think ? @galen94 @tomirisk @sanatdeshpande1